pandas/core/internals/blocks.py

```diff
@@ -1955,15 +1955,15 @@ def interpolate(self, method='pad', axis=0, inplace=False, limit=None,
                                  limit=limit),
             placement=self.mgr_locs)
 
-    def shift(self, periods, axis=0):
+    def shift(self, periods, axis=0, fill_value=None):
         """
         Shift the block by `periods`.
 
         Dispatches to underlying ExtensionArray and re-boxes in an
         ExtensionBlock.
         """
         # type: (int, Optional[BlockPlacement]) -> List[ExtensionBlock]
-        return [self.make_block_same_class(self.values.shift(periods=periods),
+        return [self.make_block_same_class(self.values.shift(periods=periods, fill_value=fill_value),
```

Comment:
Is this too wide? Check the azure output to see if the linting build failed.

Reply:
fixed